opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -15,24 +15,28 @@
 #ifndef OPENCENSUS_EXPORTERS_STATS_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 #define OPENCENSUS_EXPORTERS_STATS_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 
-#include "absl/strings/string_view.h"
+#include <string>
 
 namespace opencensus {
 namespace exporters {
 namespace stats {
 
+struct StackdriverOptions {
+  // The Stackdriver project ID to use.
+  std::string project_id;
+
+  // The opencensus_task is used to uniquely identify the task in Stackdriver.
+  // The recommended format is "{LANGUAGE}-{PID}@{HOSTNAME}". If PID is not
+  // available, a random number may be used.
+  std::string opencensus_task;
+};
+
 // Exports stats for registered views (see opencensus/stats/stats_exporter.h) to
 // Stackdriver. StackdriverExporter is thread-safe.
 class StackdriverExporter {
  public:
-  // Registers the exporter and sets the project ID and task value. project_id
-  // should be the exact id of the project, as in the GCP console, with no
-  // prefix--e.g. "sample-project-id". opencensus_task is used to uniquely
-  // identify the task in Stackdriver. The recommended format is
-  // "{LANGUAGE}-{PID}@{HOSTNAME}"; if {PID} is not available a random number
-  // may be used.
-  static void Register(absl::string_view project_id,
```

Comment:
Should we keep backwards compatibility for the API. Suggestion is to deprecate this and add a date when this will be removed.

Reply:
Done.

Reply:
I tested it and (with clang 5.0) it produces a warning like:

```
INFO: From Compiling examples/grpc/stackdriver.cc:
examples/grpc/stackdriver.cc:44:54: warning: 'Register' is deprecated: Register() without StackdriverOptions is deprecated and will be removed on or after 2019-03-20 [-Wdeprecated-declarations]
  opencensus::exporters::stats::StackdriverExporter::Register("ab", "cd");
```